demos/serverless/src/package-lock.json

```diff
@@ -1,174 +1,18 @@
 {
   "name": "amazon-chime-sdk-js-serverless-demos-lambda",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Can you upgrade NPM to 8.3.0 and then generate the package-lock? The `lockfileVersion` changes if you are on older NPM versions.